runtime/src/accounts_db.rs

```diff
@@ -6920,7 +6920,7 @@ impl AccountsDb {
         bank_hash_info.snapshot_hash
     }
 
-    pub fn update_accounts_hash(
+    pub fn update_accounts_hash_from_index(
         &self,
```

Comment:
the fact this uses index or not should be transparent. I don't think it needs to be in the fn name. Thoughts? The api just says to update the hash. Doesn't matter how it is calculated. It is only called from benches, so maybe you could put _bench or _tests in the name. This should no be used by runtime code.